src/components/navbar/navbar.js

```diff
@@ -9,16 +9,16 @@ import './navbar.scss';
 const NavBar = () => (
     <div class="w3-container w3-bar w3-black">
     <div className="w3-bar-item"><div className="logo"><img className="logo-image" src={Logo} /></div></div>
-    <div ><a href="#" class="w3-bar-item" id="nav-links">HOME</a></div>
-    <div ><a href="#" class="w3-bar-item" id="nav-links">TOURNAMENTS</a></div>
-    <div ><a href="#" class="w3-bar-item" id="nav-links">MATCH SCHEDULES</a></div>
-    <div ><a href="#" class="w3-bar-item" id="nav-links">NEWS</a></div>
-    <div ><a href="#" class="w3-bar-item" id="nav-links">ABOUT</a></div>
-    <div><a href="#" class="w3-bar-item w3-button w3-right" id="nav-button">SIGN IN</a></div>
-    <div><a href="#" class="w3-bar-item w3-right"><img src={Instagram} /></a></div>
-    <div><a href="#" class="w3-bar-item w3-right"><img src={Twitter} /></a></div>
-    <div><a href="#" class="w3-bar-item w3-right"><img src={Youtube} /></a></div>
-    <div><a href="#" class="w3-bar-item w3-right"><img src={Facebook} /></a></div>
+    <div ><a href="#" className="w3-bar-item" id="nav-links">HOME</a></div>
+    <div ><a href="#" className="w3-bar-item" id="nav-links">TOURNAMENTS</a></div>
+    <div ><a href="#" className="w3-bar-item" id="nav-links">MATCH SCHEDULES</a></div>
+    <div ><a href="#" className="w3-bar-item" id="nav-links">NEWS</a></div>
+    <div ><a href="#" className="w3-bar-item" id="nav-links">ABOUT</a></div>
+    <div><a href="#" className="w3-bar-item w3-button w3-right" id="nav-button">SIGN IN</a></div>
+    <div><a href="#" className="w3-bar-item w3-right"><img src={Instagram} /></a></div>
+    <div><a href="#" className="w3-bar-item w3-right"><img src={Twitter} /></a></div>
+    <div><a href="#" className="w3-bar-item w3-right"><img src={Youtube} /></a></div>
+    <div><a href="#" className="w3-bar-item w3-right"><img src={Facebook} /></a></div>
```

Comment:
Irene, please review this area. I advise pulling first, some changes were made in the navbar. Also, Sammad has done the routing, so you can just adjust, so the sign in button in the navbar links to your sign-in/sign-up page